src/frontend/mosh-server.cc

```diff
@@ -769,14 +767,25 @@ static void serve( int host_fd, Terminal::Complete &terminal, ServerConnection &
 	  if ( !network.shutdown_in_progress() ) {
 	    network.set_current_state( terminal );
 	  }
-	  
-	  #ifdef HAVE_UTEMPTER
-	  /* update utmp entry if we have become "connected" */
-	  if ( (!connected_utmp)
+      #if defined(HAVE_SYSLOG) || defined(HAVE_UPTEMPTER)
+      #ifdef HAVE_UTEMPTER
+      if (!connected_utmp) {
```

Comment:
There are indentation problems here (and the next hunk).  Mosh source code is generally written in GNU style, using both spaces, and tabs at 8-character stops for indentation.  This code looks like it was edited with 4-character tab stops.